CHANGELOG.md

```diff
@@ -1,87 +1,101 @@
+# CHANGELOG
```

Comment:
I have the urge to fix yellow squigglies 
![image](https://user-images.githubusercontent.com/77301861/214430102-b9b73e5f-7d14-4bde-b45a-ede874b02532.png)